### PR TITLE
extinguishers are actually good at doing their job now

### PR DESCRIPTION
**Pull request by Spacedong**
wow gee they actually do shit now

fixes issue #3292

removes memechem from them, they're now 100% water

### CODE REVIEW

**Comment by banthebantz** [comment]
this does now btfo any and all firestacks

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm [hunk 1]
```diff
@@ -175,8 +175,7 @@
 	if(!istype(M, /mob/living))
 		return
 	if(method == TOUCH)
-		M.adjust_fire_stacks(-(reac_volume))
-	..()
```
Comment (ghost) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm:179]: Why did you remove the ..() call?

Comment (Spacedong) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm]: the world may never know

**Comment by Spacedong** [comment]
@banthebantz it only btfos fire on mobs, extinguishing turfs is the same

**Comment by banthebantz** [comment]
This is shitty though, now literally a 1u water paper cup can extinguish any and all fire which makes fire stack chems like napalm useless. You've massively nerfed fuel chems and fire in general and made cryogenic fluid worse at its intended purpose than water. 

What we all wanted was for extinguishers to be better than rolling not for water to be buffed so massively, literally just replace water in extinguishers with cryogenic fluid and its done you can extinguish fires decently including turf fires (which are not affected by this pr at all)  without ruining balance.

**Comment by Spacedong** [comment]
im not replacing water with your memechems you kike

**Comment by banthebantz** [comment]
>didn't even refute my arguments

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm [hunk 2]
```diff
@@ -175,8 +175,7 @@
...
+		M.ExtinguishMob()
```
Comment (JohnGinnane) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm:178]: This is too overpowered. I think the best thing is to multiply the reac_volume by 2 or 3 so that water is much more effective

File: code/game/objects/items/weapons/extinguisher.dm [hunk 3]
```diff
@@ -39,8 +39,7 @@
 
 /obj/item/weapon/extinguisher/New()
 	create_reagents(max_chem)
-	reagents.add_reagent("water", 45)
-	reagents.add_reagent("cryogenic_fluid", 5)
+	reagents.add_reagent("water", 50)
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/extinguisher.dm:42]: This is okay

**Comment by JohnGinnane** [comment]
This PR should buff it against turf fires too because it's almost impossible to combat fires with extinguishers
